userdocs/src/usage/container-runtime.md

```diff
@@ -1,5 +1,10 @@
 # Define Container Runtime
 
+!!! warning
+    Starting with Kubernetes version `1.24`, dockershim support has been deprecated. Therefore, if you create a cluster using `eksctl` on version `1.24` or higher, the information below no longer applies, and the only suppoted container runtime is `containerd`. Trying to set it otherwise will return a validation error.
```

Comment:
```suggestion
    Starting with Kubernetes version `1.24`, dockershim support has been deprecated. Therefore, if you create a cluster using `eksctl` on version `1.24` or higher, the information below no longer applies, and the only supported container runtime is `containerd`. Trying to set it otherwise will return a validation error.
```